rpcs3/Emu/Cell/lv2/lv2.cpp

```diff
@@ -1965,14 +2005,29 @@ bool lv2_obj::wait_timeout(u64 usec, ppu_thread* cpu, bool scale, bool is_usleep
 			if (remaining > host_min_quantum)
 			{
 #ifdef __linux__
-				// Do not wait for the last quantum to avoid loss of accuracy
-				wait_for(remaining - ((remaining % host_min_quantum) + host_min_quantum));
+				// With timerslack set low, Linux is precise for all values above 10us
```

Comment:
The "hack" on line 1991 is now unnecessary with timerslack. It should just be 10 at all times.